geemap/map_widgets.py

```diff
@@ -377,3 +380,183 @@ def _objects_info(self, latlon):
                     nodes.append(tree_node)
 
         return self._root_node("Objects", nodes)
+
+
+class DrawActions(enum.StrEnum):
```

Comment:
Thanks a lot for your work on this. Some CI tests failed because of `enum.StrEnum`, which seems only available in Python 3.11. We need an alternative to support Python 3.8-3.10

Reply:
Reference: https://stackoverflow.com/questions/75850073/python-enum-attributeerror-module-enum-has-no-attribute-strenum

Reply:
Ah, thanks for catching this so quickly, working on a fix now. And I found one other issue related to `clear_draw_control` keyword didn't get properly initialized on the reset() function.

Reply:
The ipyleaflet DrawControl is quite buggy. Here are some issues I am aware of. 

- https://github.com/jupyter-widgets/ipyleaflet/issues/1119
- https://github.com/jupyter-widgets/ipyleaflet/issues/876
- https://github.com/jupyter-widgets/ipyleaflet/issues/849
- https://github.com/jupyter-widgets/ipyleaflet/issues/599

Reply:
I've amended the [last commit](https://github.com/gee-community/geemap/pull/1666/commits/7c0d85a471a0d4bf743d606d1862257091a77ced) to use the normal `enum.Enum`, so hopefully the tests will not break there.